src/compiler/compile/render_ssr/handlers/Element.ts

```diff
@@ -44,6 +44,18 @@ export default function(node: Element, renderer: Renderer, options: RenderOption
 		class_expression_list.length > 0 &&
 		class_expression_list.reduce((lhs, rhs) => x`${lhs} + ' ' + ${rhs}`);
 
+	const style_expression_list = node.styles.map(style_directive => {
+		const { name, expression: { node: expression }, text } = style_directive;
+		if (text) {
+			return x`"${name}: ${text};"`;
+		}
+		return x`"${name}: " + ${expression} + ";"`;
+	});
+	
+	const style_expression = 
+		style_expression_list.length > 0 && 
+		style_expression_list.reduce((lhs, rhs) => x`${lhs} + ' ' + ${rhs}`);
+
```

Comment:
Basically just cribbing from the styles code above — this whole file could use a lot of DRY cleanup